default/stringtables/en.txt

```diff
@@ -6572,6 +6572,12 @@ SITREP_TECH_RESEARCHED
 SITREP_TECH_RESEARCHED_LABEL
 Tech Researched
 
+SITREP_EMPIRE_TECH_RESEARCHED_DETECTION
+The %empire% [[encyclopedia DETECTION_TITLE]] has upgraded to %rawtext:dstrength%
+
+SITREP_EMPIRE_TECH_RESEARCHED_DETECTION_LABEL
+Enemy Tech Upgrade
```

Comment:
Should probably specify _Detection_ Tech Upgrade

Reply:
The question here is if other things should have the same label (as it is used to filter sitreps).
I am not sure though if the label name or the label text is used for grouping/filtering (probably the label name).

Reply:
One way or the other, the label name should match the label text.  Either both should be particular to detection, or neither should.

Reply:
If it was the label text the grouping could be the translators decision